blog/2023/12-11-roku-v200/index.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Roku Version 2.0.0
+description: I Want a Rokupotamus for Christmas
+authors: 1hitsong
+date: 2023-12-11
+slug: roku-200
+tags: [release, roku]
+---
+
+When we say this 2.0.0 release is a major milestone, we honestly mean it. New features and bug fixes abound, but even behind the scenes the updates continue all the way to the foundation with us updating every single file in the app to support a new programming language. You can't get more monumental than that!
+
+Read more to learn about some of the exciting new features.
+
+\- 1hitsong
+
+{/* truncate */}
+
+Requires minimum server version of 10.8.1
```

Comment:
I don't think this is really worth mentioning. 10.8.1 was released a year ago. If you do want to mention it I'd move it to the "downloads" part.